plyer/facades/notification.py

```diff
@@ -2,8 +2,8 @@ class Notification(object):
     '''Notification facade.
     '''
 
-    def notify(self, title='', message='', app_name='', app_icon='',
-                timeout=10):
+    def notify(self, title='', message='', ticker='', app_name='', app_icon='',
+               timeout=10):
```

Comment:
Adding a new keyword argument should happen at the end of the argument list, or people using them with direct values ("app name" instead of app_name="app name") will get a breakage.